File: test/py/case81_inheritance.py
# このファイルは `test/py/case81_inheritance.py` のテスト/実装コードです。
# 役割が分かりやすいように、読み手向けの説明コメントを付与しています。
# 変更時は、既存仕様との整合性とテスト結果を必ず確認してください。

class Base81:
    def value(self) -> int:
        return 81


class Child81(Base81):
    def value2(self) -> int:
        return self.value() + 1


if __name__ == "__main__":
    c: Child81 = Child81()
    print(c.value2())

File: test/cs/case81_inheritance.cs
// このファイルは `test/cs/case81_inheritance.cs` の変換結果（C#サンプル）です。
// Python入力との対応関係を追いやすくするため、日本語コメントを付与しています。
// 仕様変更時は、対応する Python 側ケースと合わせて更新してください。

using System;

public static class Program
{
    public class Base81
    {
        public int value()
        {
            return 81;
        }
    }

    public class Child81 : Base81
    {
        public int value2()
        {
            return (this.value() + 1);
        }
    }

    public static void Main(string[] args)
    {
        Child81 c = new Child81();
        Console.WriteLine(c.value2());
    }
}
